nnstreamer_example/custom_example_tensorrt/nnstreamer_customfilter_example_tensorrt_reshape.cc

```diff
@@ -0,0 +1,484 @@
+/**
+ * NNStreamer TensorRT Custom Filter Example: Reshape
+ *
+ * Copyright (C) 2020 Dongju Chae <dongju.chae@samsung.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * @file  nnstreamer_customfilter_example_tensorrt_reshape.cc
+ * @date  14 Sep 2020
+ * @brief  TensorRT Custom NNStreamer Filter Example: Reshape
+ * @author  Dongju Chae <dongju.chae@samsung.com>
+ * @bug  No known bugs
+ *
+ * Reference code: sampleDynamicReshape.cpp in TensorRT samples
+ *
+ * This example reshapes the inputs to the given dimensions.
+ * The custom property is to be given as, "custom=D1:D2:D3"
+ * E.g., custom=224:224:3
+ *
+ */
+
+#include <glib.h>
+#include <string.h>
+#include <tensor_filter_custom.h>
+#include <nnstreamer_plugin_api.h>
+#include <nnstreamer_log.h>
+
+#include <iostream>
+#include <memory>
+#include <vector>
+
+#include <NvInfer.h>
+#include <NvInferRuntimeCommon.h>
+#include <cuda_runtime_api.h>
+
+#ifdef USE_UNIFIED_MEMORY
```

Comment:
This is to be determined in run-time...

Reply:
Yes, I'm revising the codes to get extra options.